### ISSUE TITLE
Updated yelp-api for iOS 10 and Swift 3

### ISSUE THREAD

**Issue by dtnguy72** [Yelp/yelp-api#176]
Hello,

It there a way to get the updated version for the yelp api for the current iOS 10 and Swift 3 update?

Thanks

**Comment by mittonk** [comment]
@dtnguy72 : Are you hitting issues using the current version?

**Comment by dtnguy72** [comment]
Hello,

Yes, I am just having some trouble converting it to the latest version of Swift.

**Comment by virgytam** [comment]
Hi @dtnguy72, what is the specific issue you're facing? Error messages would be very helpful.

**Comment by dtnguy72** [comment]
Hello,

I just take a screenshot of the error messages I am receiving. I tried reinstalling the pod but get the same errors.

**Comment by virgytam** [comment]
Can you share the screenshot or the error message?

**Comment by dtnguy72** [comment]
Hello,

Here is the screenshot in the png file attachment

**Comment by mittonk** [comment]
@dtnguy72 : Looks like you're trying to send your screenshot via email --- I don't think that's going to work.

Instead, try logging into github.com and following:
* https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/

**Comment by dtnguy72** [comment]
<img width="1280" alt="screen shot 2016-10-10 at 4 00 18 pm" src="https://cloud.githubusercontent.com/assets/21061299/19251471/07c26cd4-8f05-11e6-98e2-90b63cfecf8b.png">


Hopefully you can read this now. 
![image](https://cloud.githubusercontent.com/assets/21061299/19251487/1c6aa12e-8f05-11e6-8d1c-b7aab14b32a2.png)

**Comment by virgytam** [comment]
just wanted to respond back that we can definitely see the attachment now. currently looking into the issue

**Comment by dtnguy72** [comment]
Great!

Thanks so much for your help!

**Comment by virgytam** [comment]
Hi @dtnguy72, can you tell us what steps you took to reinstall the CocoaPod? 

I found a similar issue on StackOverflow here: http://stackoverflow.com/questions/17072396/cocoapods-errors-on-project-build and the top response was to remove the workspace file and then run `pod install` again. Maybe give that a shot first?

**Comment by dtnguy72** [comment]
I tried reinstalling the pod again and opened the workspace file successfully without any errors. However when I tried running the project, the build successed but the simulator did not appear. 

Re-creating CocoaPods due to major version update.
Analyzing dependencies
Downloading dependencies
Installing OCMock (3.1.5)
Installing OHHTTPStubs (4.6.0)
Installing OMGHTTPURLRQ (3.1.1)
Installing TDOAuth (1.1.2)
Generating Pods project
Integrating client project
Pod installation complete! There are 3 dependencies from the Podfile and 4 total
pods installed.
